src/ds323x/alarms.rs

```diff
@@ -127,6 +133,19 @@ where
         self.iface.write_data(&mut data)
     }
 
+    /// Set Alarm1 for a time every day.
```

Comment:
Could you add a description of the matching strategy (i.e. fires when h, m, s match)